_data/simple-icons.json

```
@@ -3324,6 +3324,11 @@
             "hex": "F01F7A",
             "source": "https://codecov.io"
         },
+        {
+            "title": "CodeCrafters",
+            "hex": "00CEDE",
+            "source": "https://github.com/codecrafters-io/frontend/blob/2f15115b73843fea57a412ce243ff1cedb5e69f7/public/assets/images/logo/outline-color.svg"
```

```suggestion
            "source": "https://github.com/codecrafters-io/frontend/blob/2f15115b73843fea57a412ce243ff1cedb5e69f7/public/assets/images/logo/logomark-color.svg"
```
My opinion is that the other logo is more appropriate since it used in their website too.